util/src/main/java/io/camunda/zeebe/util/sched/Actor.java

```diff
@@ -119,7 +122,7 @@ protected void handleFailure(final Throwable failure) {
   }
 
   public void onActorFailed() {
-    // clean ups
+    closeFuture.complete(null);
```

Comment:
The same here as with `onActorClosed()`, this introduces a potential point of failure, if someone forgets to call `super.onActorFailed()` when overwriting this method.